src/Uno.UI.RuntimeTests/Common.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project>
+
```

Comment:
Consider making this a `Directory.Build.props`, and add the following import.

```csharp
  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
```

Then individual projects don't have to have an import.

Reply:
Indeed. Thanks!